package.json

```diff
@@ -25,7 +25,7 @@
     "polymer-cli": "^1.9.5",
     "wct-browser-legacy": "^1.0.1"
   },
-  "version": "",
+  "version": "2.1.0",
```

Comment:
So far we haven't populated this field because we hadn't decided whether we were going to publish to NPM.  What's the rationale for adding it to this one?

Note: if the script is rerun for this, this change would be lost.